en/developer_guide/custom_content_attribute_display.rst

```diff
@@ -0,0 +1,50 @@
+Customize Content Attribute Display
+===================================
+
+Context
+-------
+
+In order to have minimum constraints on contents, Open Orchestra allows you to store any type
+of ``value`` in the ``ContentAttribute``.
```

Comment:
I think the introduction still lacks some context. Maybe it should talk about contents, reintroducing that contents have several attributes, whose values are stored in ContentAttribute. And then it can talk about the values types.